seleniumByTechnocredits/src/seleniumPractise/prachiJ/Assingment1/AlertDemo_Q1.java

```diff
@@ -0,0 +1,53 @@
+package seleniumPractise.prachiJ.Assingment1;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import util.Init;
+
+/*
+ * Alert Demo > Submit > Alert
+ * Validation of the values of Alert Demo with of the Alert.
+ */
+public class AlertDemo_Q1 {
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		String url = "http://automationbykrishna.com/#";
+		driver.get(url);
+		Thread.sleep(2000);
+		driver.findElement(By.xpath("//*[@class='nav navbar-nav navbar-right']/preceding-sibling::ul/li[4]/a")).click();
+		Thread.sleep(2000);	
+
+		AlertDemo_Q1 alertDemo_Q1 = new AlertDemo_Q1();
+		alertDemo_Q1.alertValuesValidation(driver);
+	}
+	
+	void alertValuesValidation(WebDriver driver) throws InterruptedException
+	{
+		String firstName = "Prachi";
+		String lastName = "Jain";
+		String compName = "CG";
+		WebElement e1 = driver.findElement(By.xpath("//input[@id='UserFirstName']"));
+		WebElement e2 = driver.findElement(By.xpath("//input[@id='UserLastName']"));
+		WebElement e3 = driver.findElement(By.xpath("//input[@id='UserCompanyName']"));	
+		e1.sendKeys(firstName);
+		e2.sendKeys(lastName);
+		e3.sendKeys(compName);
+		String fName = e1.getAttribute("value");
```

Comment:
Prachi, if you are sending firstname, lastname , company in send keys, then you already have values. So no need to again  do element.getattribute("value"). Here you can avoid redundancy.
If you are directly sending values in .sendkeys() then it is appropriate to right getAttribute()

Reply:
Ammended suggested changes